dockerfiles/idp_deploy.Dockerfile

```diff
@@ -103,7 +105,7 @@ COPY keys.example $RAILS_ROOT/keys
 COPY public/ban-robots.txt $RAILS_ROOT/public/robots.txt
 
 # Copy application.yml.default to application.yml
-COPY ./config/application.yml.default.prod $RAILS_ROOT/config/application.yml
+COPY ./config/application.yml.default.k8s_deploy $RAILS_ROOT/config/application.yml
```

Comment:
On 2nd thoughts, is it useful to have `default` in the file name?

Reply:
I think for now it's ok, at least while waiting for a resolution of the issue you mentioned above.